e2e/testdata/Dockerfile.notary-server

```diff
@@ -0,0 +1,4 @@
+ARG NOTARY_VERSION=0.5.0
```

Comment:
Why `0.5.0` here ? `ARG     NOTARY_VERSION=v0.6.0`, it was `0.6.0` before :stuck_out_tongue_closed_eyes: 
We should even bump to `v0.6.1`

Reply:
latest version on the hub is 0.5.0 https://hub.docker.com/_/notary/


cc @endophage

Reply:
ohhh :sweat:

Reply:
Yes, see https://github.com/docker/cli/pull/1064#discussion_r189012761